ambry-commons/src/main/java/com.github.ambry.commons/HelixNotifier.java

```diff
@@ -64,21 +64,23 @@
 
   /**
    * A constructor that gets a {@link HelixNotifier} based on {@link HelixPropertyStoreConfig}.
+   * @param zkServers the ZooKeeper server address.
    * @param storeConfig A {@link HelixPropertyStore} used to instantiate a {@link HelixNotifier}. Cannot be {@code null}.
    */
-  public HelixNotifier(HelixPropertyStoreConfig storeConfig) {
+  public HelixNotifier(String zkServers, HelixPropertyStoreConfig storeConfig) {
```

Comment:
should this be `zkClientConnectString`?

Reply:
The string specifies the zookeeper server address, I copied it from `ZkClient` ctor. Since it is only used for `ZKClient` instantiation. I think it might be ok, what do you think of it?